### ISSUE TITLE
Some letters are not displayed

### ISSUE THREAD

**Issue by letsevi** [mozilla/pdf.js#11561]
Attach (recommended) or Link to PDF file here: [pdf_first.pdf](https://github.com/mozilla/pdf.js/files/4147517/pdf_first.pdf)

Configuration:
- Web browser and its version: **Google Chrome 79.0.3945.130**
- Operating system and its version: **Windows 7 Professional SP 1**
- PDF.js version: **2.2.228, also reproduce on 2.4.326 (official Demo)**
- Is a browser extension: **No**

What is the expected behavior? (add screenshot)
![image](https://user-images.githubusercontent.com/8323814/73648498-bf7a8700-468e-11ea-9559-794fa8f60393.png)

What went wrong? (add screenshot)
![image](https://user-images.githubusercontent.com/8323814/73648595-f5b80680-468e-11ea-894b-22da533012bf.png)

**Comment by Snuffleupagus** [comment]
From the console:
```
PDF ceb35e3efe02cd6958db69532f054686 [1.7 itext-paulo-155 (itextpdf.sf.net-lowagie.com) / pdftk 2.02 - www.pdftk.com] (PDF.js: 2.4.264) viewer.js:1229:15
Warning: TT: nested IDEFs not allowed pdf.worker.js:1004:13
Warning: TT: ENDF bad stack pdf.worker.js:1004:13
Warning: TT: undefined function: 2 pdf.worker.js:1004:13
Warning: TT: complementing a missing function tail pdf.worker.js:1004:13
Warning: TT: undefined function: 2 pdf.worker.js:1004:13
```

**Comment by Snuffleupagus** [comment]
Fixed by PR #12186.
